### ISSUE TITLE
Remove non-USDR users/agencies from USDR tenant

### ISSUE THREAD

**Issue by mattbroussard** [usdigitalresponse/usdr-gost#359]
In GOST prod, there are a bunch of users and agencies in the USDR tenant (ID `1`) that don't belong there. We are going to give special privileges to this tenant (namely: viewing the list of all tenants and creating new tenants), so we need to move these users to a demo tenant or state-specific tenants as appropriate.

To ensure consistency, this should probably be done programmatically with a single-use Knex seed file, rather than manually updating the DB. Note that after ARPA Reporter merge happens, there is the potential for other tables with association to tenant ID 1 (or users/agencies therein) to also need adjustment.

Once this is done, the `usdigitalresponse.org` email check in `isUSDRSuperAdmin` can potentially be removed.

This is a followup task from #189

**Comment by mhuang-usdr** [comment]
duplicate of #356. I was just going to do this via the UI to save eng effort.

**Comment by mattbroussard** [comment]
You can't do this via the UI. There is (by design) no in-product way to move users/agencies across tenant boundary

**Comment by mhuang-usdr** [comment]
I have an admin account in the DEMO tenant. I was going to delete all the existing users and recreate them. Will that not work?

**Comment by mattbroussard** [comment]
Sorry for late response; yes, you can delete and re-create users in the UI, you just cannot _move_ users from one tenant to another.

Note that pre-merge ARPA Reporter did not support deletion of users and has tables with foreign key to `users.id` that may cause user deletion in GOST to fail if the user has interacted with ARPA Reporter in certain ways (such as creating uploads or certifying a reporting period). This is the same reason the importer tool needed to remap duplicate emails rather than just delete those users.
